modular-docs-manual/content/topics/using_text_snippets_or_text_fragments.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+// Base the file name and the ID on the module title. For example:
+// * file name: my-concept-module-a.adoc
+// * ID: [id='my-concept-module-a']
+// * Title: = My concept module A
+
+// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
+[id='using_text_snippets_or_text_fragments']
+// The `context` attribute enables module reuse. Every module's ID includes {context}, which ensures that the module has a unique ID even if it is reused multiple times in a guide.
+= Text snippets or text fragments (pseudo-modules)
+//In the title of concept modules, include nouns or noun phrases that are used in the body text. This helps readers and search engines find the information quickly.
+//Do not start the title of concept modules with a verb. See also _Wording of headings_ in _The IBM Style Guide_.
+
+[NOTE]
+The following standard is recommended when the documentation is being maintained without a Content Management System (CMS) capable of managing complex interrelations between modules.
+
+The use of reusable text snippet files (or text fragment files) is discouraged due to the complications that can arise due to the complexity they introduce.
+
+Snippet (fragment) file use should be limited to:
+
+* Standardized admonitions (such as 'Technology preview' and 'Beta' text).
+* Where there is an existing standard between the upstream and downstream communities.
+
+
+//.Additional resources
```

Comment:
@tylerkelly13  looks good, just wondering about these commented items?

Reply:
I used the concept module template for this section and commented out the unnecessary stuff in case it is needed later.